### PR TITLE
HS-1350 Add FlowKafka Integration Test for MetricProcessor

### PR DESCRIPTION
**Pull request by FrancescoBuc86**
https://opennms.atlassian.net/browse/HS-1350

### CODE REVIEW

**Review by freddy61025 — APPROVED** [review]
LGTM. Just a very minor extra import.

File: metrics-processor/main/src/test/java/org/opennms/horizon/metrics/FlowProcessorIntegrationTest.java [hunk 1]
```diff
@@ -57,14 +62,14 @@
...
+    public static final GrpcCleanupRule grpcCleanupRule = new GrpcCleanupRule();
 
-    @Autowired
-    public GrpcIngesterMockServer grpcIngesterMockServer;
+    //@Autowired
+    public static GrpcIngesterMockServer grpcIngesterMockServer = new GrpcIngesterMockServer();
 
-    @Autowired
-    public GrpcInventoryMockServer grpcInventoryMockServer;
+    //@Autowired
```
Comment (freddy61025) [inline metrics-processor/main/src/test/java/org/opennms/horizon/metrics/FlowProcessorIntegrationTest.java:71]: remove comments.

**Review by freddy61025 — APPROVED** [review]
LGTM just some minor cleanup.

File: metrics-processor/main/src/test/java/org/opennms/horizon/metrics/FlowProcessorIntegrationTest.java (hunk 1 above)
Comment (FrancescoBuc86) [reply metrics-processor/main/src/test/java/org/opennms/horizon/metrics/FlowProcessorIntegrationTest.java]: done

File: metrics-processor/main/src/test/java/org/opennms/horizon/flows/FlowProcessorTest.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
 
 package org.opennms.horizon.flows;
 
+import org.junit.Ignore;
```
[the remark this answers is not in the document]
Comment (FrancescoBuc86) [reply metrics-processor/main/src/test/java/org/opennms/horizon/flows/FlowProcessorTest.java:31]: removed it

**Pull request merged**